lib/pronto/flay.rb

```diff
@@ -4,20 +4,27 @@
 module Pronto
   class Flay < Runner
     def run
-      files = ruby_patches.map(&:new_file_full_path)
-      files = ::Flay.filter_files(files)
-
       if files.any?
-        flay.process(*files)
         flay.analyze
         messages
       else
         []
       end
     end
 
+    # The current Flay (2.8.0) takes care of filtering
+    # files by looking at .flayignore.
+    # Saving the returned Flay object at @flay so we
+    # can inspect it and build the messages Array.
+    #
+    # Coercing files with Array to protect against
+    # +nil+ values.
     def flay
-      @flay ||= ::Flay.new
+      @flay ||= ::Flay.run(Array(files))
+    end
+
+    def files
+      @files ||= ruby_patches.map(&:new_file_full_path)
```

Comment:
Hi Ryan, you could also be nicknamed Zeneagle (with eagle's eye).
You're great! 👏 

I really don't remember why I have put this Array call, so I think you're right.
There's no need for it.
Thank you for spotting it